app/src/main/java/com/github/orkest/View/auth/SignIn.kt

```diff
@@ -131,7 +131,7 @@ fun signIn(activityResultLauncher: ActivityResultLauncher<Intent>,
 private fun updateUI(user: FirebaseUser?, navController: NavController) {
     if (user != null) {
         Log.d(TAG, "User is not null")
-        navController.navigate("signup")
+//        navController.navigate("signup")
```

Comment:
Why did you comment this line?

Reply:
Did this to fix a bug, but @saraanej found the solution